src/App.js

```diff
@@ -0,0 +1,74 @@
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import testImg from './assets/images/test-img.png';
+import Photos from './pages/Photos';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import NoResult from './components/NoResult';
+import Loader from './components/Loader';
+
+const API_URL = 'https://jsonplaceholder.typicode.com/albums/1/photos';
```

Comment:
Maybe let's change it to const API_URL = 'https://jsonplaceholder.typicode.com'; and add specific routes when making request. It would be good to move it to new file, because it will also be used by other components.